### PR TITLE
fix(DIA-67): do not perform image resize in HeroRails

### PR DESCRIPTION
**Pull request by araujobarret**
This PR resolves [DIA-67] 

### Description

Looks like `Image` by default performs a resize in Gemini impacting the quality(check screenshots), the image becomes very blurry. This PR avoids the resize and improves the quality of the `HeroUnitsRail`.

|  **Before**  | **After**   |
|---|---|
| |  |

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- improve hero rails image quality - araujobarret

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[DIA-67]: https://artsyproduct.atlassian.net/browse/DIA-67?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by gkartalis — APPROVED** [review]
looks good, maybe we could update the PR title since we are not using opaqueImageView

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (improve hero rails image quality - araujobarret)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 12e06a4f454a879c6eb5da78566d9ed11c8fa6b4
</p>

**Comment by jonallured** [comment]
Love it - thanks for this @araujobarret !! 🙌

File: src/app/Scenes/Home/Components/HeroUnitsRail.tsx [hunk 1]
```diff
@@ -38,7 +38,7 @@ const HeroUnit: React.FC<HeroUnitProps> = ({ item }) => {
   return (
     <Touchable key={item.internalID} onPress={handlePress}>
       <Flex bg="black100" flexDirection="row" height={CARD_HEIGHT} width={screenWidth}>
-        <Image height={CARD_HEIGHT} src={imageSrc} width={cardImageWidth} />
+        <Image height={CARD_HEIGHT} src={imageSrc} width={cardImageWidth} performResize={false} />
```
Comment (damassi) [inline src/app/Scenes/Home/Components/HeroUnitsRail.tsx:41]: @araujobarret - what is the tradeoff here? Without performing the resize, a super large image is fetched. Can we look at the underlying implementation and perhaps adjust there?

**Pull request merged**